spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -897,3 +979,13 @@ deployer.<app>.kubernetes.imagePullPolicy=Always
 
 Replacing `<app>` with the name of your app and the desired Image Pull Policy.
 
+An Image Pull Policy can be configured at the server level in the container env section of a deployment YAML, for example:
+
+```
+        env:
+        - name: SPRING_CLOUD_DEPLOYER_KUBERNETES_DEPLOYMENT_IMAGE_PULL_POLICY
+          value: Always
+```
+
+Replacing `Always` with the desired Image Pull Policy.
```

Comment:
Should we include the link to the enum class to highlight the available options?

Reply:
na i think for here the docs should be updated when the enum gets updated to include descriptions. linking the to the `KubernetesDeployerProperties` in the start of the docs i think is OK as there are many more properties than currently outlined